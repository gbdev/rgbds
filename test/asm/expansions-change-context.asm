def string_start equs "println \"hello"
string_start world"

def triple_string_start equs "println \"\"\"multi"
triple_string_start;ple
line
strings"""

def cond_start equs "if 0\nprintln \"false\"\nelif"
cond_start 1
  println "true"
else
  println "nan"
endc

def loop_start equs "rept 3\nprintln"
loop_start "lol"
endr

def macro_start equs "macro foo\nprintln"
macro_start \1
endm
  foo 42
