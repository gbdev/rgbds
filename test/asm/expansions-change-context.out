hello world
multi;ple
line
strings
true
lol
lol
lol
$2A
